server/fleet/hosts.go

```diff
@@ -154,6 +180,7 @@ type HostListOptions struct {
 	LowDiskSpaceFilter *int
 }
 
+// TODO(Sarah): Are we missing any filters here? Should all MDM filters be included?
```

Comment:
I'm not quite sure why we even have this vs doing `hlo == HostListOptions{}` to check if it is empty (i.e. the zero value of the struct). But yeah, if it's actually used somewhere I guess all fields should be there, but I'd do the zero-value check instead to be honest (maybe keep the `Empty()` method just to make it nicer and clearer, but internally do a zero-value check).

Reply:
Ahhh yeah that makes sense.